src/Nokia_LCD.cpp

```diff
@@ -29,6 +29,19 @@ Nokia_LCD::Nokia_LCD(const uint8_t clk_pin, const uint8_t din_pin,
       kDc_pin{dc_pin},
       kCe_pin{ce_pin},
       kRst_pin{rst_pin},
+      kBl_pin{NULL},
+      mX_cursor{0},
+      mY_cursor{0} {}
+
+Nokia_LCD::Nokia_LCD(const uint8_t clk_pin, const uint8_t din_pin,
+                     const uint8_t dc_pin, const uint8_t ce_pin,
+                     const uint8_t rst_pin, const uint8_t bl_pin)
+    : kClk_pin{clk_pin},
+      kDin_pin{din_pin},
+      kDc_pin{dc_pin},
+      kCe_pin{ce_pin},
+      kRst_pin{rst_pin},
+      kBl_pin{bl_pin},
```

Comment:
Instead of creating an overloaded constructor a default argument should be used instead. Less code.